src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -1116,6 +1119,7 @@ ucs_status_t uct_rc_mlx5_ep_handle_failure(uct_rc_mlx5_ep_t *ep,
        is not updated for the error cqe and all outstanding wqes*/
     rc_iface->tx.cq_available += ep->tx.wq.bb_max -
                                  uct_rc_txqp_available(&ep->super.txqp);
+    ep->super.flags           &= ~UCT_RC_EP_FLAG_CONNECTED;
```

Comment:
does it mean flush on failed ep is also not allowed?
@Artemy-Mellanox are we posting NOP/RDMA_WRITE(0) in case of lush(CANCEL)? we should not.

Reply:
than we may stuck in a state when last non-signaled operation performed before we moved QP to error. Or do I miss something?

Reply:
ok, removed flag reset

Reply:
https://github.com/openucx/ucx/pull/5795#pullrequestreview-532409051